report-results/example-queries/index.md

```diff
@@ -209,6 +209,69 @@ result
 }
 ```
 
+## Query Report Containing Multiple Linked Assets
```

Comment:
Please add the new fields to the master query (instead of creating a new sample query)

Reply:
Now I see maybe you did this already, if so, let's delete this "extra" query. Let me know if I am misunderstanding something.

Reply:
@tabhus you want to leave this in?